### ISSUE TITLE
Exception in thread "main" java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by anayyar82** [databricks/spark-avro#150]
Exception in thread "main" java.lang.NoSuchMethodError: org.apache.spark.sql.DataFrameReader.load(Ljava/lang/String;)Lorg/apache/spark/sql/Dataset;
	at com.sabre.sparkavro.SparkAvroReader$delayedInit$body.apply(SparkAvroReader.scala:25)
	at scala.Function0$class.apply$mcV$sp(Function0.scala:40)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App$$anonfun$main$1.apply(App.scala:71)
	at scala.App$$anonfun$main$1.apply(App.scala:71)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:32)
	at scala.App$class.main(App.scala:71)

**Comment by anayyar82** [comment]
Here is the code : 

object SparkAvroReader extends App {

  private val conf = ConfigFactory.load()

  private val sparkConf = new SparkConf().
    setAppName("Spark Parquet").
    setMaster("Avro Reader").
    set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")

  val sc = new SparkContext()

  val sqlContext = new SQLContext(sc)
  import sqlContext.implicits._

  val df = sqlContext.read
    .format("com.databricks.spark.avro")
    .load("/user/hive/warehouse/shoprecord/year=2016/month=06/day=01/*.avro")

  df.groupBy($"request.posPCC", $"request.invokedservice", $"Request.posCityCode",
    $"request.hostedAirline", $"request.requestorID", $"request.requestorType",
    $"request.requestType", $"responseitineraries".getItem(0).getField("Cache")
      .getItem(0).getField("id")).count().show()

}

**Comment by JoshRosen** [comment]
Which versions of Spark and `spark-avro` are you using? I'm wondering whether you're using incompatible versions.

**Comment by VeenitShah** [comment]
@JoshRosen I think I faced the same issue with different version of spark, spark-avro and scala. It's confusing for new users. After I've cloned the `spark-avro` repo to my local pc, if I wanted to build spark-avro `2.0.1` version for Scala 2.11, what changes do I need to make? Maybe @anayyar82 can follow the same guideline and try to run it again with a different version.

The documentation states that I need to use `libraryDependencies += "com.databricks" %% "spark-avro" % "2.0.1"` which I cannot understand at this point in time.

Your comments would be really helpful. Thanks in advance.

Regards,
V
